package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "impact-development",
+  "version": "2.0.0",
+  "homepage": "https://impactdevelopment.github.io",
+  "private": true,
+  "devDependencies": {
+    "eslint-plugin-react-hooks": "^1.6.0",
+    "raw-loader": "^3.0.0",
+    "react-scripts": "3.0.1",
+    "react-snap": "^1.23.0",
+    "stylelint": "^10.0.1",
+    "stylelint-config-recommended": "^2.2.0",
+    "stylelint-config-styled-components": "^0.1.1",
+    "stylelint-processor-styled-components": "^1.8.0"
+  },
+  "dependencies": {
+    "color": "^3.1.2",
+    "markdown-to-jsx": "^6.10.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-helmet-async": "^1.0.2",
+    "react-router-dom": "^5.0.1",
+    "react-transition-group": "^4.1.0",
+    "styled-components": "^4.3.1"
+  },
+  "scripts": {
+    "start": "set PORT=80 && react-scripts start",
+    "build": "react-scripts build",
+    "postbuild": "react-snap",
+    "test": "react-scripts test --env=jsdom --watchAll",
+    "eject": "react-scripts eject"
+  },
+  "reactSnap": {
+    "include": [
+      "/changelog",
+      "/donate"
+    ],
+    "crawl": true,
+    "skipThirdPartyRequests": true,
+    "headless": false
+  },
+  "browserslist": [
+    ">1%",
+    "not dead",
+    "not ie 11",
+    "not op_mini all"
+  ]
+}
```

Comment:
After experimenting with a build/deploy workflow in #18 I've settled on using github-page's build in integration with [Jekyll](https://jekyllrb.com/) instead. Unfortunately I can't get behind using react for this.